src/foam/dao/ORBitalDAO.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao',
+  name: 'ORBitalDAO',
+  documentation : 'DAO class used to delegate a method execution request to the specified object (specified by an ORBRequest object).',
+  extends: 'foam.dao.ProxyDAO',
+  flags: ['java'],
+
+  javaImports: [
+    'foam.core.FObject',
+    'foam.core.ClassInfo',
+    'foam.core.MethodInfo',
+    'foam.core.X',
+    'foam.nanos.logger.Logger'
+  ],
+
+  properties: [
+
+  ],
+
+  methods: [
+    {
+      name: 'cmd_',
+      type: 'Object',
+
+      args: [
+        { name: 'x' , type: 'foam.core.X'}, { name: 'obj' , type: 'Object' }
+      ],
+
+      javaCode: `
+        ORBRequest request = null;
+        if ( obj instanceof ORBRequest ) request = (ORBRequest) obj;
```

Comment:
Please don't format if/else this way in general, but in this case, this would be easier to read:

 if ( ! ( obj instanceof ORBRequest )  ) return getDelegate().cmd_(x, obj);

request = (ORBRequest) obj;

Reply:
ok